src/openrct2/ride/Ride.cpp

```diff
@@ -1304,6 +1313,480 @@ std::optional<CoordsXYZ> sub_6C683D(
     return retCoordsXYZ;
 }
 
+/**
+ *
+ *  rct2: 0x006CA162
+ */
+money32 place_provisional_track_piece(
```

Comment:
These functions do not belong here. They are used only by the ui project and therefore if they are going to move they should move to the ui project. Its hard to see what you have changed as you have moved and I think modified after the move. Just do one thing.